### ISSUE TITLE
Track the number of poapedus minted by the user

### ISSUE THREAD

**Issue by Randomacy** [Poapedu/poapedu-app#12]
Ensure that we display the number of poapedus by the user correctly on the page

**Comment by harishkotra** [comment]
@Randomacy Do we need an API to store the poapedus minted by the user? If yes, can you share the payload here so that I can create the API endpoint and share it back with you to store a poapedu once it is minted?

Thank you

**Comment by Randomacy** [comment]
You're right, we should store at least something about the user's poapedus once it's minted.
